### DISCUSSION TITLE
Remote GUI starts services which it should not

### DISCUSSION THREAD

**Discussion by txr13** [Chia-Network/chia-blockchain#10939] [Support]
I am currently running the following setup:

Machine 1: Linux, full node, wallet, farmer
Machine 2: Windows, GUI
Machines 3-5: Various, harvesters

All machines are running version 1.3.1 and have full connectivity to each other. The GUI on machine 2 connects to the full node on machine 1. All harvesters on machines 3 through 5 show up correctly as remote harvesters connected to the farmer on machine 1.

I am running the GUI in farming mode so that I have full visibility into my setup, rather than just the wallet. There are no plots on machine 1, and I have tried to stop the harvester process there because it is extraneous. However, I find that when I shut down the harvester on machine 1, the GUI on machine 2 returns a message "Starting harvester" and the harvester process on machine 1 respawns.

Is there a config option I can set to prevent the GUI from spawning processes when connecting to a remote daemon?

**Comment by loppefaaret** [comment]
Are you using the remote UI as described here:
https://github.com/Chia-Network/chia-blockchain/wiki/Connecting-the-UI-to-a-remote-daemon ?

**Reply by txr13** [reply]
That is correct. I followed that article exactly in order to set it up with machines 1 and 2.

**Reply by loppefaaret** [reply]
thank you, im honestly a bit surprised it is starting services on the remote host - i dont believe that is intended
let me do a little poking around and see what i can figure out

**Reply by txr13** [reply]
Just a status update: everything has been updated to 1.3.2. I did not start the harvester on machine 1, but when the GUI was launched on machine 2, it went through launching everything on the remote daemon. The farmer, node, and wallet processes were already running, so those message flew past all at once, but it definitely took the time to start the harvester, as it paused on "Starting harvester" again (and the harvester process launched on machine 1).

I realize there were no changes in 1.3.2 other than the bump for OpenSSL, so I wasn't expecting any changes to the behavior I described earlier. This is more just confirming that the behavior still exists and wasn't a one-off occurrence.

**Reply by loppefaaret** [reply]
yeah, thing is that the work done to provide the remote UI, was mostly done by a 3rd party volunteer, so none of our engineers "just knows"... it is something that needs a bit more digging, but thank you very much for keeping me honest, and up to date on this!

**Reply by txr13** [reply]
Just a status update, same behavior present in 1.3.4.

**Reply by txr13** [reply]
Same behavior present in 1.3.5.
